src/raylib.h

```diff
@@ -447,11 +448,18 @@ typedef struct Music {
     AudioStream stream;         // Audio stream
     unsigned int frameCount;    // Total number of frames (considering channels)
     bool looping;               // Music looping enable
-
+    bool background;            // Update in audio thread ?
     int ctxType;                // Type of music context (audio filetype)
     void *ctxData;              // Audio context data, depends on type
 } Music;
 
+typedef struct AudioDeviceInfo
```

Comment:
It can probably be avoided, no need of a new structure to query that info if required, probably most users won't care about it.